pootle/apps/accounts/utils.py

```diff
@@ -30,3 +34,177 @@ def get_user_by_email(email):
             return User.objects.get(email__iexact=email)
         except User.DoesNotExist:
             return None
+
+
+def merge_user(src_user, target_user):
+    """Merges one user to another.
+
+    The following are fields are updated (model: fields):
+    - units: submitted_by, commented_by, reviewed_by
+    - submissions: submitter
+    - suggestions: user, reviewer
+
+    :param src_user: `User` instance to merge from.
+    :param target_user: `User` instance to merge to.
+    """
+
+    # Move units to target_user.
```

Comment:
This doesn't *move* any units — it changes unit attribution for the denormalized unit fields.